tntcarrier.php

```diff
@@ -977,10 +977,10 @@ public function hookextraCarrier($params)
 
 		$output = null;
 		if (isset($this->context) && method_exists($this->context->controller, 'addJS'))
-		{	
+		{
 			$this->context->controller->addJS('https://maps.google.com/maps/api/js?sensor=true');
 			$this->context->controller->addJS($this->_path.'js/relais.js');
-			$this->context->controller->addJS($this->_path.'js/jquery-ui-1.8.10.custom.min.js');
+			$this->context->controller->addJS($this->_path.'js/jquery-ui.js');
```

Comment:
Could you please insert a condition to load your own jQuery UI if _PS_VERSION_ < 1.5?